payload/Library/Scripts/Install or Defer.sh

```diff
@@ -190,31 +226,61 @@ EOF
 # by previous checks).
 run_updates () {
 
-    # Display HUD with updating message.
-    "$JAMFHELPER" -windowType "hud" -windowPosition "ur" -icon "$LOGO" -title "$MSG_UPDATING_HEADING" -description "$MSG_UPDATING" -lockHUD &
+    # On Apple Silicon Macs, running softwareupdate --install via script is
+    # currently unsupported, so we'll just inform the user with a persistent
+    # alert and open the Software Update window for manual update.
+    if [[ "$PLATFORM_ARCH" = "arm64" ]]; then
 
-    # Run Apple system updates.
-    echo "Running $INSTALL_WHICH Apple system updates..."
-    # macOS Big Sur requires triggering the restart as part of the softwareupdate action, meaning the script will not be able to run its clean_up functions until the next time it is run.
-    if [[ "$OS_MAJOR" -gt 10 ]] && [[ "$INSTALL_WHICH" = "all" ]]; then
-      echo "System will restart as soon as the update is finished. Cleanup tasks will run on a subsequent update check."
-    fi
-    # shellcheck disable=SC2086
-    UPDATE_OUTPUT_CAPTURE="$(/usr/sbin/softwareupdate --install --${INSTALL_WHICH} ${RESTART_FLAG} --no-scan 2>&1)"
-    echo "Finished running Apple updates."
-
-    # Trigger restart if script found an update which requires it.
-    if [[ "$INSTALL_WHICH" = "all" ]]; then
-        # Shut down the Mac if BridgeOS received an update requiring it.
-        if [[ "$UPDATE_OUTPUT_CAPTURE" == *"select Shut Down from the Apple menu"* ]]; then
-            trigger_restart "shut down"
-        # Otherwise, restart the Mac.
-        else
-            trigger_restart "restart"
+        echo "This is an Apple Silicon Mac with pending updates. Displaying persistent alert until updates are applied..."
+
+        # Loop this check until softwareupdate --list shows no more pending
+        # recommended updates.
+        while [[ $(/usr/sbin/softwareupdate --list) == *"Recommended: YES"* ]]; do
```

Comment:
Open to discussion on possibly adding a delay to this while loop. Maybe a `sleep 60` before `killall jamfHelper`? Gives people a chance to save documents and quit apps before System Preferences reasserts itself.

Reply:
Added `sleep 60` below.